cookbook/integration_llama-index_milvus-lite.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>This example uses OpenAI for embeddings and chat completions, so you also need to specify your <a href="https://platform.openai.com/docs/api-reference/api-keys" target="_blank">OpenAI key</a> in environment variable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>